AEXS/aex-5.md

```diff
@@ -43,230 +43,145 @@ By standardization of the set of messages that should be used for inter-wallet c
 
 ## Specification
 
-### Protocol Messages
+### JSON-RPC 2.0 Methods
 
 #### General
 
-- `error`: Used to communicate any error occurred. Error code `1` to `8` are reserved and reused here from [AEX-2](https://github.com/aeternity/AEXs/blob/master/AEXS/aex-2.md#types-of-errors).
-
-- `ping/pong`: ping/pong messages for liveness check.
-
-  JSON-RPC 2.0 structure:
-
-  ```jsonc
-  Ping:
-  {
-    "jsonrpc": "2.0",
-    "method": "ping",
-    "params": {
-       "id": "<unique_identifier_uuidv4>"
-    },
-    "id": 1,
-    "version": 1
-  }
-
-  Pong:
-  {
-    "jsonrpc": "2.0",
-    "result": {
-       "id": "<unique_identifier_uuidv4>",
-       "data": "pong"
-    },
-    "id": 1,
-    "version": 1
-  }
-  ```
+- `error`: Used to communicate any error occurred. Error code `1` to `9` are reserved and reused here from [AEX-2](https://github.com/aeternity/AEXs/blob/master/AEXS/aex-2.md#types-of-errors).
+
+- `ping/pong`: general ping/pong messages to check liveness. Implementation of this method is not mandatory for wallets communicating over a transport layer that have native support for liveness check.
+
+    ##### Parameters
+
+    `Object`
+
+    - `id` - The unique identifier, must conform to the [UUID v4 standards](https://tools.ietf.org/html/rfc4122#page-14)
```

Comment:
Saying »A unique identifier« sounds better than »The unique identifier«, I think